README.md

```diff
@@ -58,11 +58,17 @@ First, make sure there is no other Postgres instance running locally, or using p
 Quickly start a Postgres database in docker with
 
 ```
-docker run --name naisten_linja_db -e POSTGRES_USER=nl_dev -e POSTGRES_PASSWORD=nl_dev -e POSTGRES_DB=nl_dev -v ./dbdata:/var/lib/postgresql/data -p 5432:5432 postgres:11.9
```

Comment:
Oh shoot, really sorry. This command does not work for me neither because of the relative path `./dbdata` . Was copy-pasting this from my project note and realized I forgot to fix it after writing that.

The updated command works for me! Thank you! : )

Reply:
No worries, it is totally understandable! And I think every time someone tests the documentation, there is something to add. Wether it is something like this, or they use Windows and then there are some unexpected things happening 😄 (had this experience once)

Yay, great! 🎉